example/ssd/dataset/pycocotools/coco.py

```diff
@@ -55,12 +55,12 @@
 # from . import mask as maskUtils
 import os
 from collections import defaultdict
-import sys
-PYTHON_VERSION = sys.version_info[0]
-if PYTHON_VERSION == 2:
-    from urllib import urlretrieve
-elif PYTHON_VERSION == 3:
+from six.moves import string_types
```

Comment:
I think six is existing dependency of mxnet. For sring_types you can use mxnet.base.string_types

Reply:
This file does not import mxnet.

Reply:
I am trying to follow the advise in https://github.com/apache/incubator-mxnet/pull/10833#issuecomment-404568638

Reply:
I agree with #10833, since mxnet is always a dependency, so I guess import it and use mxnet.base.string_types is better, otherwise users will complain it six is not installed.